esphome/components/switch/switch.cpp

```diff
@@ -20,6 +20,11 @@ void Switch::toggle() {
   ESP_LOGD(TAG, "'%s' Toggling %s.", this->get_name().c_str(), this->state ? "OFF" : "ON");
```

Comment:
Since changes are being made, might as well update this one
```suggestion
  ESP_LOGD(TAG, "'%s' Toggling %s.", this->get_name().c_str(), ONOFF(this->state));
```